pipelines/mitx/mfe-frontend-app-learning.yml

```diff
@@ -1,65 +1,68 @@
 ---
```

Comment:
Should this file be renamed to mitxonline?

Reply:
Possibly. Eventually this will be a template pipeline so it might be more accurate to rename it to `edx`